addons/binding/org.openhab.binding.rfxcom.test/src/test/java/org/openhab/binding/rfxcom/internal/messages/RFXComInterfaceControlMessageTest.java

```diff
@@ -0,0 +1,211 @@
+/**
+ * Copyright (c) 2010-2017 by the respective copyright holders.
+ *
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ */
+package org.openhab.binding.rfxcom.internal.messages;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceMessage.TransceiverType._433_92MHZ_TRANSCEIVER;
+
+import javax.xml.bind.DatatypeConverter;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.openhab.binding.rfxcom.internal.config.RFXComBridgeConfiguration;
+import org.openhab.binding.rfxcom.internal.exceptions.RFXComException;
+import org.openhab.binding.rfxcom.internal.messages.RFXComInterfaceControlMessage;
+
+/**
+ * Test for RFXCom-binding
+ *
+ * @author Mike Jagdis
+ * @since 2.0.0
+ */
+public class RFXComInterfaceControlMessageTest {
+    private RFXComBridgeConfiguration configuration = new RFXComBridgeConfiguration();
+
```

Comment:
There is one test case described in the manual can you validate it and maybe include it as a test:

> Display un-decoded messages
> Protocols Enabled: La Crosse, Oregon, AC, ARC and X10
> 0D 00 00 00 03 53 00 80 08 27 00 00 00 00